adapters/relevantdigital/relevantdigitaltest/exemplary/simple-audio.json

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
`static/bidder-info/relevantdigital.yaml` doesnot mentions audio as supported type for app and site